### DISCUSSION TITLE
hey guys i have an problem in creating the custom network in metamask for anvil even after giving the correct RPC and chain id: could any one pls help me out this?

### DISCUSSION THREAD

**Discussion by SanthiyaMeganathan** [Cyfrin/foundry-full-course-cu#2502] [Q&A]
![Screenshot 2024-09-18 181145](https://github.com/user-attachments/assets/cd8f6a84-5564-4b17-809f-aa0ccb33cfed)
could any one please help me out this?

**Comment by EngrPips** [comment]
Hello @SanthiyaMeganathan, Did you try to keep an `Anvil` chain running when adding it to `Metamask`?
